### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by minwoox**
- Brave 5.10.1 -> 5.10.2
- bucket4j-core 4.9.0 -> 4.10.0
- Curator 4.2.0 -> 4.3.0
- java-jwt 3.10.0 -> 3.10.1
- jetty-alpn-agent 2.0.9 -> 2.10.0
- json-unit 2.14.0 -> 2.16.0
- JUnit 5.6.0 -> 5.6.1
- Micrometer 1.3.5 -> 1.4.0
- Netty 4.1.37.Final -> 4.1.48.Final
  - netty-tcnative-boringssl-static 2.0.29.Final -> 2.0.30.Final
- Opensaml 3.4.3 -> 3.4.5
- Reactor 3.3.3.RELEASE -> 3.3.4.RELEASE
- Retrofit 2.7.2 -> 2.8.0
- RxJava 3.0.0 -> 3.0.1
- Spring2.1
  - 2.1.12.RELEASE -> 2.1.13.RELEASE
- Tomcat 9.0.31 -> 9.0.33
  - Tomcat8.5 8.5.51 -> 8.5.53
- Zookeeper 3.5.7 -> 3.6.0
- Build
  - java-support 7.5.0 -> 7.5.1
  - kotlin-allopen 1.3.70 -> 1.3.71
  - Gradle 6.2.2 -> 6.3.0

### CODE REVIEW

**Comment by minwoox** [comment]
I will send the followup PR to switch using JDK 14.

File: dependencies.yml [hunk 1]
```diff
@@ -301,18 +301,18 @@ net.javacrumbs.future-converter:
...
 net.shibboleth.utilities:
-  java-support: { version: '7.5.0' }
+  java-support: { version: '7.5.1' }
 
 org.apache.curator:
   curator-recipes:
-    version: '4.2.0'
+    version: '4.3.0'
     javadocs:
     - https://static.javadoc.io/org.apache.curator/curator-recipes/4.2.0/
```
Comment (ikhoon) [inline dependencies.yml]:
```suggestion
    - https://static.javadoc.io/org.apache.curator/curator-recipes/4.3.0/
```

File: dependencies.yml [hunk 2]
```diff
@@ -421,7 +421,7 @@ org.jctools:
 # If you want to change `org.jetbrains.kotlin:kotlin-allopen` version, 
 # you also need to change `org.jetbrains.kotlin.jvm` version in `build.gradle`.
 org.jetbrains.kotlin:
-  kotlin-allopen: { version: &KOTLIN_VERSION '1.3.70' }
+  kotlin-allopen: { version: &KOTLIN_VERSION '1.3.71' }
```
Comment (ikhoon) [inline dependencies.yml]: Don't we need to update `build.gradle`?

Comment (minwoox) [reply dependencies.yml]: Oops, thanks!

File: dependencies.yml (hunk 1 above)
Comment (minwoox) [reply dependencies.yml]: Fixed. 😉

**Pull request merged**